.github/dockerfiles/ov_build/ubuntu_22_04_x64/Dockerfile

```diff
@@ -58,15 +51,16 @@ RUN mkdir ${SCCACHE_HOME} && cd ${SCCACHE_HOME} && \
     curl -SLO https://github.com/mozilla/sccache/releases/download/${SCCACHE_VERSION}/${SCCACHE_ARCHIVE} && \
     tar -xzf ${SCCACHE_ARCHIVE} --strip-components=1 && rm ${SCCACHE_ARCHIVE}
 
+ENV PATH="$SCCACHE_HOME:$PATH"
 # Setup pip
 ENV PIP_VERSION="24.0"
 RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    python3.8 get-pip.py  --no-cache-dir pip==${PIP_VERSION} && \
+    python3 get-pip.py --no-cache-dir pip==${PIP_VERSION} && \
     python3.11 get-pip.py --no-cache-dir pip==${PIP_VERSION} && \
     rm -f get-pip.py
 
-# Use Python 3.11 as default instead of Python 3.8
-# Using venv here 'cause other methods to switch the default Python on Ubuntu 20 break both system and wheels build
+# Use Python 3.11 as default
+# Using venv here 'cause other methods to switch the default Python on Ubuntu break both system and wheels build
 RUN python3.11 -m venv venv
```

Comment:
do we need if we install one python version only

Reply:
Both system (3.12) and Python 3.11 are installed, to not break tests that are hardcoded to use 3.11, the `venv` is created.